rolling/distribution.yaml

```diff
@@ -3060,6 +3060,21 @@ repositories:
       url: https://github.com/ros-drivers/phidgets_drivers.git
       version: rolling
     status: maintained
+  pick_ik:
+    doc:
+      type: git
+      url: https://github.com/PickNikRobotics/pick_ik.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/PickNikRobotics/pick_ik-release.git
```

Comment:
@tylerjw IIRC proper Rolling release requires a release repository under the `ros2-gbp` organization. You may want to do the same for the Humble release (https://github.com/ros/rosdistro/pull/35592), just for the sake of convenience. CC @nuclearsandwich.

Reply:
I'm happy to put this in a different release repo. I was not aware of this rule. Let me know if you'd like me to move this?

Reply:
@tylerjw yes please. See https://www.ros.org/reps/rep-2002.html#managing-build-farm-writable-release-repositories for the rationale and check https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html on joining existing release teams to get your `ros2-gbp` release repo.

Reply:
I created this issue to have new release repos created for my three current projects that are not part of ros2-gbp and have them moved to the moveit release team: https://github.com/ros2-gbp/ros2-gbp-github-org/issues/163

Reply:
@hidmic Friendly ping on this. I create the issue over on ros2-gbp to request the release repos be created so I can migrate over there. Can you approve my request and create the new repos?